### ISSUE TITLE
Menu - Focus indicator accessibility (Grommet theme)

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7470]
The Menu component's focus indicator on the menu items gets cut off so that instead of a border around the component it appears as a line under the component. The focus indicator should be consistent across components and display as at least a 2px thick perimeter around the focused component.

<img width="184" alt="Image" src="https://github.com/user-attachments/assets/92dcf6fd-3181-4420-917f-b96c9b5c4166" />

In the Grommet theme we don't show the focus indicator at all on the menu

<img width="226" alt="Image" src="https://github.com/user-attachments/assets/c00c115c-40c8-447d-a6a7-e9c229679751" />

Relevant WCAG rules:
https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html
https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance

**Comment by jcfilben** [comment]
Resolved in hpe theme but still an issue for grommet theme
